moveit_ros/planning/introspection/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_executable(moveit_list_planning_adapter_plugins src/list_planning_adapter_plugins.cpp)
+ament_target_dependencies(moveit_list_planning_adapter_plugins
+  Boost
```

Comment:
I don't see any boost references in `list_planning_adapter_plugins.cpp`.

Reply:
Good catch